site/tests/integration/accountpage.test.ts

```diff
@@ -0,0 +1,211 @@
+
+  await expect(page.locator('h3:has-text("Alice")')).toBeVisible();
+
+  await expect(page.locator("text=@alice")).toBeVisible();
+
+  for (const [testId, href] of [
+    ["overview", "/@alice"],
+    ["blocks", "/@alice/blocks"],
+    ["schemas", "/@alice/schemas"],
+  ] as const) {
+    const item = page.locator(`[data-testid='profile-page-${testId}-tab']`);
+    await expect(item).toBeVisible();
+    await expect(item).toHaveAttribute("href", href);
+  }
+
+  // Overview tab tests
+  await page.locator("[data-testid='profile-page-overview-tab']").click();
+
+  await expect(page).toHaveURL("/@alice");
+
+  await expect(
+    page.locator("text=You haven’t created any blocks or schemas yet"),
+  ).toBeVisible();
+
+  await expect(
+    page.locator("text=Start building to see your creations show up here."),
+  ).toBeVisible();
+
+  await expect(page.locator("text=Build a block")).toBeVisible();
+
+  await expect(page.locator("text=Build a block")).toHaveAttribute(
+    "href",
+    "/docs/developing-blocks",
+  );
+
+  await expect(page.locator("text=Create a schema")).toBeVisible();
+
+  // Blocks tab tests
+  await page.locator("[data-testid='profile-page-blocks-tab']").click();
+
+  await expect(page).toHaveURL("/@alice/blocks");
+
+  await page.locator("text=You haven’t created any blocks yet").click();
+
+  await expect(
+    page.locator("text=Start building to see your creations show up here."),
+  ).toBeVisible();
+
+  await expect(page.locator("text=Build a block")).toBeVisible();
+
+  await expect(page.locator("text=Build a block")).toHaveAttribute(
+    "href",
+    "/docs/developing-blocks",
+  );
+
+  // Schema tab tests
+  await page.locator("[data-testid='profile-page-schemas-tab']").click();
+
+  await expect(
+    page.locator("text=You haven’t created any schemas yet"),
+  ).toBeVisible();
+
+  await expect(
+    page.locator("text=Start building to see your creations show up here."),
+  ).toBeVisible();
+
+  await expect(page.locator("text=Create a schema")).toBeVisible();
+});
+
+const codeBlockMetadata = (blocksData as ExpandedBlockMetadata[]).find(
+  ({ name }) => name === "@hashintel/block-code",
+);
+
+if (!codeBlockMetadata) {
+  throw new Error("Code block should be prepared before tests are run");
+}
+
+test("key elements should be present when guest user views account page", async ({
+  page,
+  isMobile,
+}) => {
+  await page.goto("/@hash");
+  await expect(page.locator('h3:has-text("HASH")')).toBeVisible();
+  await expect(page.locator("text=@hash")).toBeVisible();
+
+  for (const [testId, href] of [
+    ["overview", "/@hash"],
+    ["blocks", "/@hash/blocks"],
+    ["schemas", "/@hash/schemas"],
+  ] as const) {
+    const item = page.locator(`[data-testid='profile-page-${testId}-tab']`);
+    await expect(item).toBeVisible();
+    await expect(item).toHaveAttribute("href", href);
+  }
+
+  expect(
+    await page.locator("[data-testid='overview-card']").count(),
+  ).toBeGreaterThan(3);
```

Comment:
TODO: use poll

Reply:
Added in [33301ee](https://github.com/blockprotocol/blockprotocol/pull/456/commits/33301ee41c157457768abd4ea917cb38f5225dab)